Workbooks/TSG/Performance Counter TSG/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
+    "name":"Performance Counter TSG",
+    "author": "Microsoft",
+    "galleries": [{ "type": "tsg", "resourceType": "microsoft.insights/components", "order": 100 }]
```

Comment:
Unrelated to this PR: can I not specify different 'UI groups' for the same template under different galleries?

Reply:
Not sure what you are asking for.. Can we take it offline?